deploy-manage/production-guidance/kibana-traffic-scaling-considerations.md

```diff
@@ -0,0 +1,112 @@
+
+Load balancing helps to mitigate traffic spikes by horizontally scaling your {{kib}} deployments and improving {{kib}}'s availability. To learn more about load balancing, refer to [](./kibana-load-balance-traffic.md).
+::::
+
+## Before you start [_before_sizing_kibana]
+
+{{es}} is the search engine and backing database of {{kib}}. Any performance issues in {{es}} will manifest in {{kib}}. Additionally, while Elastic tries to mitigate this possibility, {{kib}} may be sending requests to {{es}} that degrade performance if {{es}} is underprovisioned.
+
+### Is the {{es}} cluster correctly sized?
+
+Follow [the production guidance for {{es}}](./elasticsearch-in-production-environments.md).
+
+### What requests is {{kib}} sending to {{es}}?
+
+In user interfaces like Dashboards or Discover, you can view the full query that {{kib}} is sending to {{es}}. This is a good way to get an idea of the volume of data and work a {{kib}} visualization or dashboard is creating for {{es}}. Dashboards with many visualizations will generate higher load for {{es}} and {{kib}}.
+
+## Basic scaling using number of concurrent users
+
+Follow this strategy if you know the maximum number of expected concurrent users.
+
+Start {{kib}} on **1 vCPU** and **2GB** of memory. This should comfortably serve a set of 10 concurrent users performing analytics activities like browsing dashboards.
+
+If you are experiencing performance issues, you can scale {{kib}} vertically by adding the following resources for every 10 additional concurrent users:
+* 1 vCPU
+* 2GB of memory
+
+These amounts are a safe minimum to ensure that {{kib}} is not resource-starved for common analytics use cases.
+
+It is recommended to scale vertically to a maximum of **8.4 vCPU** and **8GB** of memory.
+
+You should also combine vertical scaling with horizontal scaling to handle greater concurrency or bursty traffic. Refer to [](./kibana-load-balance-traffic.md) for guidance.
+
+::::{important}
+This advice does not apply to scaling {{kib}} for task manager. If you intend to use {{kib}} alerting capabilities, see [](./kibana-task-manager-scaling-considerations.md).
+::::
+
+### Scaling examples
+
+| Concurrent users | Total vCPU | Total memory | ECH and ECE deployment examples |
+| --- | --- | --- | --- |
+| 50 | 5 vCPU | 10GB | • {{kib}} instance of 8.4 vCPU and 16GB memory in 1 availability zone (creates 2 8GB nodes per availability zone)<br><br>• {{kib}} instance of 8.4 vCPU and 8GB memory each across 2 availability zones<br><br>• {{kib}} instances of 8.4 vCPU and 4GB memory each across 3 availability zones |
+| 100 | 10 vCPU | 20GB | • {{kib}} instance of 12 vCPU and 24GB memory in 1 availability zone (creates 3 8GB nodes per availability zone)<br><br>• {{kib}} instance of 8.4 vCPU and 8GB memory each across 3 availability zones<br><br>|
```

Comment:
```suggestion
| 100 | 10 vCPU | 20GB | • {{kib}} size per zone of 24 GB RAM and 12 vCPU in 1 availability zone (creates 3 x 8GB nodes)<br><br>• {{kib}} size per zone of 8GB RAM and up to 8 vCPU across 3 availability zones<br><br>|
```

Nits to bring this slightly closer to the copy on the UI, e.g. listing memory first and calling it "RAM", using 8vCPU instead of 8.4, using "size per zone" instead of "instance".